provider-ci/internal/pkg/templates/bridged-provider/.github/workflows/prerequisites.yml

```
@@ -1,3 +1,18 @@
+name: "Prerequisites"
+
+on:
+  workflow_call:
+    inputs:
+      is_pr:
+        type: boolean
+      is_automated:
+        type: boolean
+      default_branch:
+        type: string
```

Can we mark these as required?
```suggestion
        type: boolean
        required: true
      is_automated:
        type: boolean
        required: true
      default_branch:
        type: string
        required: true
```